elliott/elliottlib/bzutil.py

```diff
@@ -214,6 +214,7 @@ def is_invalid_tracker_bug(self):
         return has_keywords or has_cve_in_summary
 
     def all_advisory_ids(self):
+        errata.sync_bugzilla_bug(self.id)
```

Comment:
Think this is not needed. We only have flaw bugs in bugzilla, and these do not get this treatment.

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/1063/commits/2e297444788301fb704f7cf0fcad88cceccfaef0